frameworks/gradle/src/test/kotlin/expektTests.kt

```diff
@@ -0,0 +1,14 @@
+import org.junit.Test
+import com.winterbe.expekt.expect
+import com.winterbe.expekt.should
+
+class ExpektTests {
+  @Test
+  fun canUseExpect() {
+    expect(1).to.equal(1)
+  }
+  @Test
+  fun canUseShould() {
+    1.should.equal(1)
```

Comment:
Woah, this is amazing. I didn't realize someone replicated Chai for Kotlin!

What's really amazing is [just how clean and readable the code is](https://github.com/winterbe/expekt), even though it's statically and strongly typed. 😃